client/src/modules/stock/exit/modals/findPatient.modal.html

```diff
@@ -41,6 +42,21 @@
           disabled="!$ctrl.selected.uuid">
         </bh-find-invoice>
       </div>
+
+      <div ng-if="$ctrl.patientInvoices.length && $ctrl.joinInvoice">
+        <div ng-repeat="invoice in $ctrl.patientInvoices" class="radio">
+          <label>
+            <input
+              type="radio"
+              ng-click="$ctrl.findDetailInvoice(invoice)"
+              name="invoice">
+            <span><strong>{{ invoice.reference }}</strong> - {{ invoice.date | date:'dd MMMM yyyy, hh:mm' }}</span>
+          </label>
+        </div>
+      </div>
+      <div class="text-warning" ng-if="!$ctrl.patientInvoices.length && $ctrl.joinInvoice">
+        <i class="fa fa-warning"></i> <span translate> FORM.INFO.THIS_PATIENT_NOT_BILLED </span>
```

Comment:
FORM.INFO.THIS_PATIENT_NOT_BILLED does not appear to be translated into English.

Reply:
I just added it, for the first time I didn't save the file

Reply:
The latest update fixed this.